### DISCUSSION TITLE
"Move buttons to the right of Hamburger Menu" CSS Snippet stopped working

### DISCUSSION THREAD

**Discussion by aphirst** [piroor/treestyletab#3672] [Q&A]
Windows 10, Firefox 133

Since updating Firefox, I'm no longer able to move the buttons to the right of the hamburger menu (or rather, move the hamburger menu to the left of the buttons).

The two snippets I'm using are "Hide horizontal tabs at the top of the window" and "Move Minimize/Restore/Close window buttons to the right of Hamburger menu (after hiding horizontal tabs)" only.

![image](https://github.com/user-attachments/assets/5a8f49d4-cafd-4337-a81f-89647ca4f8e9)

I am able to hide the horizontal tabs, but when I try to do the buttons hamburger menu shift, they refuse to coexist on the same line. It worked fine on the previous FF version, and the error appeared after an update. I've tried tweaking a bunch of the variables in the CSS as well as a few of the different ways of hiding the horizontal tabs, but I just can't seem to merge them.

I'd appreciate any and all feedback on this matter, and ideally, if we can find a solution it'd be great if it could be shared and integrated back into the otherwise invaluable CSS Snippet resource.

**Comment by Supr-A** [comment]
Any progress on this matter?

**Comment by irvinm** [comment]
You could use my addon (https://github.com/irvinm/Toggle-Native-Tab-Bar#how-to-use-toggle-native-tab-bar) or just borrow the CSS outlined in the instructions.  (Just remove the `[titlepreface*=" "]` from the CSS.

**Reply by irvinm** [reply]
^--- @Supr-A

**Reply by Supr-A** [reply]
@irvinm It works! Thank you!
@piroor Please update [this](https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#move-minimizerestoreclose-window-buttons-to-the-right-of-hamburger-menu-after-hiding-horizontal-tabs)

**Reply by piroor** [reply]
@Supr-A I don't maintain CSS code snippets actively - I hope it is maintained by people who really using them (because I don't use CSS customization including userChrome.css around tab bar.)
